source/FreeRTOS_DHCPv6.c

```diff
@@ -742,23 +666,7 @@ static BaseType_t xDHCPv6ProcessEndPoint_HandleState( NetworkEndPoint_t * pxEndP
                 {
                     FreeRTOS_debug_printf( ( "vDHCPProcess: giving up %lu > %lu ticks\n", EP_DHCPData.xDHCPTxPeriod, ipconfigMAXIMUM_DISCOVER_TX_PERIOD ) );
 
-                    #if ( ipconfigDHCP_FALL_BACK_AUTO_IP != 0 )
-                        {
-                            /* Only use a fake Ack if the default IP address == 0x00
-                             * and the link local addressing is used.  Start searching
-                             * a free LinkLayer IP-address.  Next state will be
-                             * 'eGetLinkLayerAddress'. */
-                            prvPrepareLinkLayerIPLookUp( pxEndPoint );
-
-                            /* Setting an IP address manually so set to not using
-                             * leased address mode. */
-                            EP_DHCPData.eDHCPState = eGetLinkLayerAddress;
-                        }
-                    #else
-                        {
-                            xGivingUp = pdTRUE;
-                        }
-                    #endif /* ipconfigDHCP_FALL_BACK_AUTO_IP */

```

Comment:
`prvPrepareLinkLayerIPLookUp` is used for DHCPv4 to fall back IP address.
This functionality for IPv6 is not implemented.